### PR TITLE
Adjust FlattenException and HydraError normalizers to Symfony 5.4

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12                  |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| Related tickets | continuation of #14642                    |
| License         | MIT                                                          |

Some clean-ups to adjust to Symfony 5.4 standards 💃🏻.

### CODE REVIEW

File: .github/workflows/application.yml [hunk 1]
```diff
@@ -791,6 +791,15 @@ jobs:
...
                     if-no-files-found: ignore
+            
+            -
+                name: Upload logs
+                uses: actions/upload-artifact@v3
+                if: failure()
+                with:
+                    name: "Logs (non-JS, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
+                    path: var/log/
+                    if-no-files-found: ignore
```
Comment (coldic3) [inline .github/workflows/application.yml]: :+1: but not related

**Review by coldic3 — APPROVED** [review]

File: .github/workflows/application.yml (hunk 1 above)
Comment (jakubtobiasz) [reply .github/workflows/application.yml]: It was only for debugging purposes, I removed it c:.

File: src/Sylius/Behat/Context/Api/Admin/ResettingPasswordContext.php [hunk 2]
```diff
@@ -117,7 +117,7 @@ public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
 
         Assert::same($lastResponse->getStatusCode(), Response::HTTP_INTERNAL_SERVER_ERROR);
         $message = $this->responseChecker->getError($lastResponse);
-        Assert::startsWith($message, 'No user found with reset token: ');
+        Assert::startsWith($message, 'Internal Server Error');
```
Comment (jakubtobiasz) [inline src/Sylius/Behat/Context/Api/Admin/ResettingPasswordContext.php:120]: All messages for status codes > 500 are replaced by the “default” ones. Let's treat it as quick fix, I created a new task to handle this case. Let's unblock builds 🙏🏼.

File: src/Sylius/Behat/Context/Api/Shop/LoginContext.php [hunk 3]
```diff
@@ -272,7 +272,7 @@ public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
         // token is removed when used
         Assert::same($this->client->getLastResponse()->getStatusCode(), 500);
         $message = $this->responseChecker->getError($this->client->getLastResponse());
-        Assert::startsWith($message, 'No user found with reset token: ');
+        Assert::startsWith($message, 'Internal Server Error');
```
Comment (jakubtobiasz) [inline src/Sylius/Behat/Context/Api/Shop/LoginContext.php:275]: As above.
All messages for status codes > 500 are replaced by the “default” ones. Let's treat it as quick fix, I created a new task to handle this case. Let's unblock builds 🙏🏼.

**Review by NoResponseMate — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Jakub! :1st_place_medal:

**Pull request merged**